website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -181,6 +181,13 @@ resource "mongodbatlas_cloud_backup_schedule" "test" {
 * `retention_unit` - (Required) Scope of the backup policy item: days, weeks, or months.
 * `retention_value` - (Required) Value to associate with `retention_unit`.
 
+### Snapshot Distribution
+*
+* `cloud_provider` - (Required) Human-readable label that identifies the cloud provider that stores the snapshot copy. i.e. "AWS" "AZURE" "GCP"
```

Comment:
Can you add an example demonstrating how to use this feature on this page?

Reply:
Added